modules/swagger-codegen/src/main/resources/nodejs/controller.mustache

```diff
@@ -9,12 +9,23 @@ module.exports.{{nickname}} = function {{nickname}} (req, res, next) {
   {{#allParams}}
   var {{paramName}} = req.swagger.params['{{baseName}}'].value;
   {{/allParams}}
+  var responseHeaders = [];
+  var header = {};
+  var responseHeaderSchema;
+  {{#responses}}
+  {{#headers}}
+  responseHeaderSchema = {{{jsonSchema}}};
+  header.name = '{{name}}';
+  header.value = responseHeaderSchema.default;
+  responseHeaders.push(header);
+  {{/headers}}
+  {{/responses}}
   {{classname}}.{{nickname}}({{#allParams}}{{paramName}}{{#hasMore}},{{/hasMore}}{{/allParams}})
     .then(function (response) {
-      utils.writeJson(res, response);
+      utils.writeJson(res, response, null, responseHeaders);
     })
     .catch(function (response) {
-      utils.writeJson(res, response);
+      utils.writeJson(res, response, null, responseHeaders);
```

Comment:
Is it bad to pass null here for the code?

Reply:
@sandro97git I believe the null is for "arg2", which is for HTTP response code. I'm no expert in JS and I've copied the technical committee for JS to see if they've an opinion on this.

cc @CodeNinjai @frol@cliffano